stripe/src/test/java/com/stripe/android/view/PaymentMethodsAdapterTest.kt

```diff
@@ -5,6 +5,7 @@ import com.stripe.android.model.PaymentMethodFixtures
 import kotlin.test.BeforeTest
 import kotlin.test.Test
 import kotlin.test.assertEquals
+import kotlin.test.assertNull
```

Comment:
this should come after `import kotlin.test.assertNotNull`

Reply:
fixed!